### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by DirtyRacer1337**

### CODE REVIEW

File: test/namer_moviexml_test.py [hunk 1]
```diff
@@ -107,21 +105,21 @@ def test_writing_xml_metadata_genre_flag(self):
     <type>Actor</type>
     <name>Carmela Clutch</name>
     <role>Female</role>
-    <image>{fakeTPDB.get_url()}flLf1pecTlKcpJCki30l5iWXNdQ=/1000x1500/smart/filters:sharpen():upscale()/https%3A%2F%2Fcdn.metadataapi.net%2Fperformer%2F3e%2Fb6%2F0a%2F9a33557155b8a6f3e6c88bee8ed65a6%2Fposter%2F6a00376cbee2aa83c8c2514b89b332276df42c35</image>
```
Comment (4c0d3r) [inline test/namer_moviexml_test.py:110]: these urls will get called during tests -- right now I let them fail, this should stay to keep dev machines from inadvertently from calling the real tpdb instance during tests.

File: test/updatejson.sh [hunk 2]
```diff
@@ -2,9 +2,8 @@
...
-curl --request GET --get "https://api.metadataapi.net/scenes/1678283"     --header "Authorization: Bearer ${TOKEN}" --header "Content-Type: application/json"     --header "Accept: application/json" | jq > ./ea.full.json
-curl --request GET --get "https://api.metadataapi.net/scenes?q=brazzers-exxtra-suck-suck-blow"     --header "Authorization: Bearer ${TOKEN}" --header "Content-Type: application/json"     --header "Accept: application/json" | jq > ./ssb2.json
-curl --request GET --get "https://api.metadataapi.net/scenes?q=brazzers-exxtra-suck-suck-blow"     --header "Authorization: Bearer ${TOKEN}" --header "Content-Type: application/json"     --header "Accept: application/json" | jq > ./ssb2.json
-curl --request GET --get "https://api.metadataapi.net/movies?q=petite18.Harper%20Red&limit=25"     --header "Authorization: Bearer ${TOKEN}" --header "Content-Type: application/json"     --header "Accept: application/json" | jq > ./p18.json
```
Comment (4c0d3r) [inline test/updatejson.sh:10]: these are more readable by human devs comparing test failures to data with jq in the mix

Comment (DirtyRacer1337) [reply test/updatejson.sh:10]: I prefer have real server response, if you want human readable, then better use software which much faster that regular notepad for json like, [Dadroit JSON Viewer](https://dadroit.com)

File: test/namer_moviexml_test.py (hunk 1 above)
Comment (DirtyRacer1337) [reply test/namer_moviexml_test.py:110]: We don't have this images in local, besides it's from json which we get from api, so probably then better rewrite test

File: test/updatejson.sh (hunk 2 above)
Comment (4c0d3r) [reply test/updatejson.sh:10]: Fair enough, you do you :D

I'll look in to vs code json viewers

File: test/namer_moviexml_test.py (hunk 1 above)
Comment (4c0d3r) [reply test/namer_moviexml_test.py:110]: I was letting the images fail after hitting local, the test doesn't verify they get downloaded.

To be fair I had one setup to work, just to be able to verify it was hit in code coverage.   The test must have lost verification of the image download in a refactor:

https://github.com/ThePornDatabase/namer/blob/main/test/utils.py#L152

Not super worried about, just letting you know why I put these in.

Comment (DirtyRacer1337) [reply test/namer_moviexml_test.py:110]: I see, you added replace, but it just doesn't work, will work on it

**Review by 4c0d3r — APPROVED** [review]

**Comment by DirtyRacer1337** [comment]
That's why I prefer real response, jq gives different json (server made backslash escape)
Now fixed

**Pull request merged**